main.go

```diff
@@ -2,172 +2,21 @@ package main
-	}
-	bh.bot.AnswerInlineQuery(inlineConfig)
-}
-
-func (bh BotHandlers) message(update tgbotapi.Update) {
-	log.Printf("[%s] text: %s, caption: %s", update.Message.From.UserName, update.Message.Text, update.Message.Caption)
-	if update.Message.Entities != nil {
-		for _, entity := range *update.Message.Entities {
-			username := update.Message.Text[entity.Offset : entity.Offset+entity.Length]
-			if entity.Type == "mention" && username == "@reTGanslatorBot" {
-				aliases := bh.config.AllAliases()
-				for i, alias := range aliases {
-					aliases[i] = "*" + strings.ToLower(alias)
-				}
-				msg := tgbotapi.NewMessage(update.Message.Chat.ID, "Tags: "+strings.Join(aliases, " "))
-				msg.BaseChat.ReplyToMessageID = update.Message.MessageID
-				bh.bot.Send(msg)
-			}
-		}
-	}
-
-	found := false
-	for _, chat := range bh.config.Chats {
-		if update.Message.Chat.ID == chat.ID {
-			found = true
-			break
-		}
-	}
-	if !found {
-		return
-	}
-
-	for _, chat := range bh.config.Chats {
-		hasTags := false
-		for _, alias := range chat.Aliases {
-			if hasChatTag(alias, update.Message.Text) || hasChatTag(alias, update.Message.Caption) {
-				hasTags = true
-				break
-			}
-		}
-		if !hasTags {
-			continue
-		}
-
-		{
-			msg := tgbotapi.NewMessage(chat.ID, "Пересилаю повідомлення з чату "+update.Message.Chat.Title)
-			bh.bot.Send(msg)
-		}
-		if update.Message.ReplyToMessage != nil {
-			msg := tgbotapi.NewForward(chat.ID, update.Message.Chat.ID, update.Message.ReplyToMessage.MessageID)
-			bh.bot.Send(msg)
-		}
-		{
-			msg := tgbotapi.NewForward(chat.ID, update.Message.Chat.ID, update.Message.MessageID)
-			bh.bot.Send(msg)
-		}
-	}
-}
-
-func (bh BotHandlers) command(update tgbotapi.Update) {
-	aliases := bh.config.AllAliases()
-	for i := range aliases {
-		aliases[i] = "*" + strings.ToLower(aliases[i])
-	}
-	aliasesStr := strings.Join(aliases, " ")
-	contactsStr := strings.Join(bh.config.HelpContacts, " ")
-	msg := tgbotapi.NewMessage(update.Message.Chat.ID, fmt.Sprintf(`
-Щоб переслати повідомлення в інший UACT чат:
-
-(а) Ще не відправлене повідомлення
-Додайте до тексту повідомлення тег під час його написання.
-
-(б) Вже відправлене повідомлення
-Відповідайте на потрібне повідомлення і додавайте тег до тексту відповіді.
-Бот надішле обидва повідомлення в потрібний чат(и): (1) те, на яке ви відповідаєте і (2) безпосередньо вашу відповідь з тегом.
-
-Щоб побачити доступні теги, почніть писати повідомлення в будь-якому чаті UACT з @reTGanslator, і бот запропонує вам список тегів. Також можна тегнути бота у будь-якому повідомленні, і бот надішле список усіх тегів.
-Доступні такі теги:
-%s
-
-За поясненням до тегів і як працює пересилка, звертайтеся до %s
-	`, aliasesStr, contactsStr))
-	bh.bot.Send(msg)
-}
-
 func main() {
 	botToken := os.Getenv("BOT_TOKEN")
 	if botToken == "" {
 		log.Fatalf("BOT_TOKEN has to be specified")
 	}
 
-	configFile, err := os.Open("config.json")
+	configFile, err := os.Open("./config.json")
```

Comment:
config.json is inside the webhook directory right now. `go run main.go` won't work in this case

Reply:
config.json file is presented in both locations. I run it locally and it works :thinking: